queries/currency.py

```diff
@@ -35,6 +35,7 @@
 
 from query import Query, QueryStateDict
 from queries import query_json_api, iceformat_float, sing_or_plur, gen_answer
+from queries.plugins.text_to_num import *
```

Comment:
We tend to avoid import *, according to PEP guidelines. The code is more readable if all imports are explicit.

Reply:
I import the functions like this so they get called in `NonterminalNode.process` in `tree.py` (so they aren't prefixed with `text_to_num.`). Is there some cleaner way to make sure the `TöluðTala...` functions get called for each appropriate nonterminal? (Having an explicit import in this case means importing about 48 functions, which is a bit much).

I could e.g. include the `text_to_num` module in the TreeStateDict `state`, and search for the relevant function there, if it isn't found in `processor`. Or is there some way to do this with fancy imports instead?

Reply:
OK, I understand. Then the best solution is just to put a comment before the import explaining what is happening, i.e. that these are nonterminal handler functions that are deliberately being imported into the parent namespace.